openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
+    post:
+      tags:
+        - AKSK
+      summary: Create a new AKSK for a User.
+      operationId: addAKSK
+      parameters:
+        - name: project_id
+          in: path
+          required: true
+          description: AKSK to be generated for User of a Project
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AKSK'
+        '401':
+          description: Unauthorized. Token not found in header
+        '415':
+          description: Unsupported Media Type
+      requestBody:
+        $ref: '#/components/requestBodies/AKSK'
+  '/v1/{project_id}/aksks/{user_id}':
```

Comment:
could not see a resource for user? how do we get user_id?

Reply:
user_id is obtained by the GET Credential , GET Scoped Credential of the Keystone APIs. 
More details in the documentation PR : https://github.com/sodafoundation/documentation/pull/165

Reply:
As I understand from the doc , user ID is created through identity APIS , and that user id is given here, is that correct?

Reply:
yes